tools/bin/update_intellij_venv.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import os
+import xml.etree.ElementTree as ET
+
+
+def add_venv_to_xml_root(module: str, xml_root):
+    table = xml_root.find("component")
+    jdk_node = ET.SubElement(table, 'jdk', {"version": "2"})
+    ET.SubElement(jdk_node, "name", {"value": f"Python ({module})"})
+    ET.SubElement(jdk_node, "type", {"value": "Python SDK"})
+    ET.SubElement(jdk_node, "version", {"value": "Python"})
+    ET.SubElement(jdk_node, "homePath",
+                  {"value": f"$USER_HOME$/code/airbyte/airbyte-integrations/connectors/{module}/.venv/bin/python"})
+
+    roots = ET.SubElement(jdk_node, "roots")
+    annotationsPath = ET.SubElement(roots, "annotationsPath")
+    ET.SubElement(annotationsPath, "root", {"type": "composite"})
+
+    classPath = ET.SubElement(roots, "classPath")
+    classPathRoot = ET.SubElement(classPath, "root", {"type": "composite"})
+
+    ET.SubElement(classPathRoot, "root", {"url":
+                                              f'file://$USER_HOME$/code/airbyte/airbyte-integrations/connectors/{module}/.venv/lib/python3.8/site-packages',
+                                          "type": "simple"
+                                          })
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('-modules', nargs='?', help='Modules to add')
+    group.add_argument('--all-modules', nargs='?')
+
+    parser.add_argument("-input", help="Path to input jdk table")
+    parser.add_argument("-output", help="Path to output jdk table")
```

Comment:
Updated so there's nothing to specify if there's only one version of IntelliJ, and added a flag so the user can specify. I don't want to automate the selection because people might want to do this process both for pycharm and the community edition of intellij.

The script will print valid options if one is required and isn't passed

> Please select which version of Intellij to update with the `-intellij-version` flag. Options are: ['WebStorm2021.3', 'IdeaIC2021.3', 'PyCharmCE2021.3']